### DISCUSSION TITLE
JSDoc for "removeMode" is misleading

### DISCUSSION THREAD

**Discussion by jpreis** [keystonejs/keystone#7321] [Feature requests]
In the [Fields API documentation for relationships](https://keystonejs.com/docs/apis/fields#relationship), it states that the `removeMode` option controls whether to display the remove button in the UI.

The JSDoc comment however states that this option is used to control whether a related item should be deleted when the relationship is removed.

```ts
declare type CardsDisplayConfig = {
  ui?: {
    /** Determines whether removing a related item in the UI will delete or unlink it */
    removeMode?: 'disconnect' | 'none';
  }
}
```

Setting the option to `disconnect` displays the remove button in the UI wheras setting it to `none` hides it - so it works as described in the documentation page but to me it seems a bit confusing.

My suggestion would be to rename the option to something like `allowRemove` to indicate more clearly that it's just about whether or not removing is allowed.
Then, there could be an additional option to control how such a removal should be treated (unlinking vs. deleting). In that regard, I'd be glad to get some pointers as I haven't found a way to control this.

**Comment by bladey** [comment]
Hey @jpreis, thanks for the feedback, we'll take a look at this.
